ThunkLibs/Generator/analysis.h

```diff
@@ -132,7 +137,8 @@ class AnalysisAction : public clang::ASTFrontendAction {
     std::vector<ThunkedFunction> thunks;
     std::vector<ThunkedAPIFunction> thunked_api;
 
-    std::unordered_set<const clang::Type*> funcptr_types;
+    // TODO: Rename, since this is now not just per type but also per set of annotations
```

Comment:
Time to revisit this. This also at least needs a comment to explain the data structure.